src/payment-card/types.js

```diff
@@ -0,0 +1,29 @@
+/*
+Copyright (c) 2018-2019 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+
+import type {
+  InputComponentsT,
+  InputPropsT,
+  StInputPropsDiffT,
+} from '../input/types.js';
+import type {OverrideT} from '../helpers/overrides.js';
+
+export type PaymentCardComponentsT = {
+  ...InputComponentsT,
```

Comment:
fyi spreads from inexact types will mark the 'spread-ed' fields as optional [try-flow](https://flow.org/try/#0C4TwDgpgBAogHgQwMbCgXigbwD5QGYD2BAXFAM7ABOAlgHYDmU2AvgNwBQ7okUAkrREQp0WfEVIUaDKG07doAZTCUICACbxkqDJigA6A5pQAaKACMElUrQCuAWzMRKMjvKhKV6-oK0jdBvW8hYFMLKyhbBycXTiQCWgooHxRSD1UNYL8w0gBGAAZZOITUOmTgVOV0oN8dbKh8tiA)

Reply:
Interesting. Should be fine in this case, since all those props were already optional.